### DISCUSSION TITLE
Thermo LipidSearch nomenclature

### DISCUSSION THREAD

**Discussion by RobyPelle2021** [SysMedOs/LipidLynxX#20] [General]
Please note that the LipidLynxX converter does not work with the Thermo LipidSearch nomenclature system.
Lipid classes such as the following are not converted:
BisMePA - bismethyl phosphatidic acid
cPA - cyclic phosphatidic acid
dMePE - dimethylphosphatidylethanolamine
LdMePE - lysodimethylphosphatidylethanolamine
PEt - phosphatidylethanol
PMe - phosphatidylmethanol
So - sphingosine
CerG1 - hexosyl ceramide
CerG3 - trihexosyl ceramide
SoG1 - hexosyl sphingosine
ST - sulfatide
ChE - cholesterol ester
ZyE - zymosterol
AcCa - acyl carnitine
WE - wax ester
SQDG - sulfoquinovosyldiacylglycerol

**Comment by ZhixuNi** [comment]
Hi @RobyPelle2021 !

Many thanks for submitting the feedback!
Currently we are focused on improving the conversion and resource linking functions for main mammalian lipids classes.
(FA,PA, PC, PE, PG, PI, PS, SM, Cer, MG, DG, TG, ST ...)
Some lipid (sub)classes you mentioned above are currently not included due to lack of examples and well defined standard notation. 
If it is possible, can you kindly provide some examples of identifiers of following classes from LipidSearch to us, so we can have a look and implement these two classes in the next version of LipidLynxX.
```
dMePE - dimethylphosphatidylethanolamine
LdMePE - lysodimethylphosphatidylethanolamine
So - sphingosine
CerG1 - hexosyl ceramide
CerG3 - trihexosyl ceramide
SoG1 - hexosyl sphingosine
ChE - cholesterol ester
AcCa - acyl carnitine
```

When we finalized issues for main mammalian lipids classes, and received standard notation definition for other classes you mentioned (e.g. cPA, PMe), we will continue implement them in LipidLynxX.

Best wishes,

**Comment by RobyPelle2021** [comment]
Hi Zhixu Ni,

Thank you for your quick reply.

I attach a file to show you how I manually converted some lipid names that LipidLynxX skipped.
In some cases, I am not sure if the coversion is correct. In any case, Table 2 of the attached article shows the LipidSearch nomenclature.
Kind regards

Roberto


[cid:021ba0c1-fba8-4a47-8b18-dd218fce6d44]


Dott. Roberto Maria Pellegrino

Dipartimento di Chimica, Biologia e Biotecnologie
Università degli Studi di Perugia
Mail: ***@***.***

tel. 075 585 7214

sito web: www.dcbb.unipg.it

________________________________
Da: Zhixu Ni ***@***.***>
Inviato: venerdì 16 aprile 2021 11:42
A: SysMedOs/LipidLynxX ***@***.***>
Cc: Roberto Maria Pellegrino ***@***.***>; Mention ***@***.***>
Oggetto: Re: [SysMedOs/LipidLynxX] Thermo LipidSearch nomenclature (#20)


Hi @RobyPelle2021<https://github.com/RobyPelle2021> !

Many thanks for submitting the feedback!
Currently we are focused on improving the conversion and resource linking functions for main mammalian lipids classes.
(FA,PA, PC, PE, PG, PI, PS, SM, Cer, MG, DG, TG, ST ...)
Some lipid (sub)classes you mentioned above are currently not included due to lack of examples and well defined standard notation.
If it is possible, can you kindly provide some examples of identifiers of following classes from LipidSearch to us, so we can have a look and implement these two classes in the next version of LipidLynxX.

dMePE - dimethylphosphatidylethanolamine
LdMePE - lysodimethylphosphatidylethanolamine
So - sphingosine
CerG1 - hexosyl ceramide
CerG3 - trihexosyl ceramide
SoG1 - hexosyl sphingosine
ChE - cholesterol ester
AcCa - acyl carnitine


When we finalized issues for main mammalian lipids classes, and received standard notation definition for other classes you mentioned (e.g. cPA, PMe), we will continue implement them in LipidLynxX.

Best wishes,
